app/controllers/concerns/my_facilities_filtering.rb

```diff
@@ -38,11 +38,7 @@ def set_only_new_facilities
     end
 
     def facilities_by_size(facilities)
-      if (@facility_sizes - @selected_sizes).empty?
-        facilities
-      else
-        facilities.where(facility_size: @selected_sizes)
-      end
+      facilities.where(facility_size: @selected_sizes).or(facilities.where(facility_size: nil))
```

Comment:
Why grab facilities with nil sizes?

Reply:
Thanks @rsanheim! Just realizing facility size is a mandatory field in the UI. Removing this!